src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -158,7 +168,7 @@ public void Dispose()
             _parametersCache.Dispose();
         }
 
-        private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext, bool useAppContext, CancellationToken cancellationToken)
+        private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext, bool useAppContext, FeatureDefinition featureDefinition, CancellationToken cancellationToken)
```

Comment:
`string feature` shouldn't be neededs since it's part of the feature definition.